code/style.css

```diff
@@ -1,7 +1,7 @@
 /* Global css variables used for colors */
 :root {
```

Comment:
Great that you changed the background color, you could also change the text color of the value in the question to make it more visible